src/Hl7.Fhir.Base/FhirPath/FhirEvaluationContext.cs

```diff
@@ -69,7 +69,7 @@ public FhirEvaluationContext(ScopedNode node)
         /// </summary>
         public static new FhirEvaluationContext WithResourceOverrides(ITypedElement? resource, ITypedElement? rootResource = null) =>
             (FhirEvaluationContext)EvaluationContext.WithResourceOverrides(resource, rootResource);
-        public ITerminologyService? TerminologyService { get; set; }
```

Comment:
Wouldn't this hurt us when trying to implement the other terminology-related fhirpath functions except member-of?

Reply:
See discussion in #2590. I am not sure, just doing what I'm told :)